test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -25,8 +25,8 @@ images:
     converted: true
 
   centos-7:
-    filename: CentOS-7-x86_64-GenericCloud-1809.qcow2
-    url: http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1809.qcow2
+    filename: CentOS-7-x86_64-GenericCloud-2003.qcow2
+    url: http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2003.qcow2
     checksum: sha256:42c062df8a8c36991ec0282009dd52ac488461a3f7ee114fc21a765bfc2671c2
```

Comment:
This checksum is for CentOS-7-x86_64-GenericCloud-1809.qcow2.
We need to update here to `1db30c9c272fb37b00111b93dcebff16c278384755bdbe158559e9c240b73b80` if getting CentOS-7-x86_64-GenericCloud-2003.qcow2 according to http://cloud.centos.org/centos/7/images/sha256sum.txt